std_msgs/Header header
AprilTagDetection[] detections
